src/RFC-0152_EmojiId.md

```diff
@@ -83,46 +81,60 @@ list is the emoji map. For example,
 * ...
 * 🦊 => 255
 
-The emoji SHOULD be selected such that
+The emoji SHOULD be selected such that.
 
-* Similar looking emoji are excluded from the map. e.g. Neither 😁 or 😄 should be included. Similarly the Irish and
-  Côte d'Ivoirean flags look very similar, and both should be excluded.
+* Similar-looking emoji are excluded from the map. e.g. Neither 😁 or 😄 should be included. Similarly, the Irish and
+  Côte d'Ivoire flags look very similar, and both should be excluded.
 * Modified emoji (skin tones, gender modifiers) are excluded. Only the "base" emoji is considered.
 
-#### Encoding
+### Encoding
 
 The selection of an alphabet with 256 symbols means there is a direct mapping between bytes and emoji. For each byte
 in the input data to be encoded, map the byte to the corresponding emoji using the emoji map. The resulting
 concatenation of emoji characters is the emoji string.
 
-#### Emoji ID definition
+### Emoji ID definition
 
-The emoji ID is an emoji string of 33 symbols from the emoji alphabet. It uses this bitmap:
+The emoji ID is a string of 33 symbols from the emoji alphabet. It uses this bitmap:
 
 ```text
-+-----------------------------+-------------------+
-|  Node public key (256 bits) | Checksum (8 bits) |
-+-----------------------------+-------------------+
++-----------------------------+--------------------------+
+|  Node public key (256 bits) | EncodedChecksum (8 bits) |
++-----------------------------+--------------------------+
 ```
 
  The emoji ID is calculated from a node public key serialized as 32 bytes (`B`) as follows:
 
 * Use the [DammSum](https://github.com/cypherstack/dammsum) algorithm with `k = 8` and `m = 32` to compute an 8-bit
-checksum `C` from `B`.
+encoded checksum `eC` from `B`.
 * Encode `B` into an emoji string.
-* Encode `C` into an emoji string.
-* Concatenate `B` and `C` as the emoji ID.
+* Encode `eC` into an emoji string.
+* Concatenate `B` and `eC` as the emoji ID.
 
-#### Decoding
+### Checksum 
+
+For the checksum we use the [DammSum](https://github.com/cypherstack/dammsum) algorithm with `k = 8` and `m = 32` to compute an 8-bit
+encoded checksum `C` from the public key `B`.
+To verify that the network that the public key comes from is the same one, we encode the checksum with the network. 
```

Comment:
I would use a term like "assert" instead of "verify", since verification seems to imply part of the decoding and validation process.